cfme/tests/intelligence/reports/test_reports.py

```diff
@@ -669,3 +673,67 @@ def test_import_report_preserve_owner(preserve_owner, create_po_user_and_group,
 
     assert view.report_info.user.text == user
     assert view.report_info.group.text == group
+    pass
+
+
+@pytest.fixture
+def setup_vm(configure_fleecing, appliance, provider):
+    vm = appliance.collections.infra_vms.instantiate(
+        name=random_vm_name(context="report", max_length=20),
+        provider=provider,
+        template_name="env-rhel7-20-percent-full-disk-tpl",
+    )
+    vm.create_on_provider(allow_skip="default", find_in_cfme=True)
+    vm.smartstate_scan(wait_for_task_result=True)
+    yield vm
+
+    vm.cleanup_on_provider()
+
+
+@pytest.mark.tier(1)
+@pytest.mark.long_running
+@pytest.mark.customer_scenario
+@pytest.mark.provider([InfraProvider], selector=ONE_PER_CATEGORY)
+@pytest.mark.meta(automates=[1686281], server_roles=["+smartstate", "+smartproxy"])
+def test_vm_volume_free_space_less_than_20_percent(
+    appliance, setup_provider, provider, setup_vm, soft_assert
+):
+    """
+    Bugzilla:
+        1686281
+        1696420
+
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        caseimportance: medium
+        initialEstimate: 1/3h
```

Comment:
This is customer scenario. Don't you think, we have to keep it's ```caseimportance``` high?

Reply:
I think I asked this question once, and it doesn't necessarily have to be so.